prjxray/node_model.py

```diff
@@ -45,7 +45,7 @@ def __init__(
 
         self.nodes = None
 
-        self.wire_to_node_map = None
+        self.wire_to_node_map = {}
```

Comment:
Leave this as `None`.  Instead change `_build_wire_to_node_map` to initialize to the empty map.